web/README.md

```diff
@@ -0,0 +1,21 @@
+# Zone Builder web demo
+
+This is a demonstration of gluing a [Leaflet](https://leafletjs.com/)
+Javascript web map to the Rust zonebuilder crate. You can use this to draw a
+clockboard and download the GeoJSON.
+
+## Building
+
+These instructions will be improved soon. I think using npm probably simplifies
+things, and we can hook up automatic deployment to Github pages.
+
+But for now:
+
+1.  Install [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/) if you haven't before.
+
+2.  Compile the Rust code to WebAssembly. Run this from the root directory in
+    this repo: `cd ..; wasm-pack build --target web -- --features wasm`
+
+3.  Back in this directory, run `./serve_locally.py`
+
+4.  Open http://0.0.0.0:8000/ in your browser.
```

Comment:
I see this :tada: 

![image](https://user-images.githubusercontent.com/1825120/126843695-c1909c49-cc10-41a5-9596-337822d6cce6.png)